src/lib/Menu.js

```diff
@@ -86,19 +89,32 @@ const SubNavItem = styled.li`
 `;
 
 const Nav = styled.header`
+  height: 69px;
   background-color: ${style.BACKGROUND.main};
   box-shadow: ${style.SHADOW.big};
   border-radius: 5px;
   padding: 0px 40px;
 `;
 
+const MenuNavLink = NavLink.extend`
+  background: ${style.BACKGROUND.main};
+  display: ${(p) => (!p.hasSubMenu && !p.isSubMenu && p.hoverLine === 'bottom') ? 'block' : 'inline-block' };
+  height: ${(p) => (!p.hasSubMenu && !p.isSubMenu && p.hoverLine === 'bottom') ? '40px' : 'auto'};
+  border-bottom-style: ${(p) => (p.hasSubMenu && p.hoverLine === 'bottom') || p.hoverLine === 'none' ? 'none' : 'solid' };
+`;
+
 const Submenu = (props) => (
   <NavMenu>
     { props.links.map((item, index) =>
       <SubNavItem key={item.name + index}>
-        <NavLink href={item.link}>
+        <MenuNavLink
+          href={item.link}
+          hoverLine={props.hoverLine}
+          hasSubMenu={false}
+          isSubMenu={true}
+        >
```

Comment:
We probably can omit `hasSubMenu` and leave `isSubMenu` as a boolean attributes
```
  isSubMenu
>
```

Reply:
OK